packages/block-library/src/image/edit.native.js

```diff
@@ -236,13 +236,15 @@ class ImageEdit extends React.Component {
 						label={ __( 'Alt Text' ) }
 						value={ alt || '' }
 						valuePlaceholder={ __( 'None' ) }
-						separatorType={ 'fullWidth' }
-						onChange={ this.updateAlt }
+						separatorType={ 'none' }
+						onChangeValue={ this.updateAlt }
 					/>
+				</PanelBody>
+				<PanelBody padded={ false }>
```

Comment:
When I compare it with web usage of `PanelBody` this doesn't look right to me. If this was cross-platform block this code piece would produce a new expandable-collapsable section on web for `Clear All Settings` button. And 1. web doesn't have such actions, 2. AFAIU PanelBody should just be used for grouping same settings together. So I think the red action buttons on mobile don't have equivalents on web so I'd say we should define a different container or component for the action buttons, that would render nothing on web. @koke wdyt? That way we can also apply same styles to them without needing to handle separately.

<img width="387" alt="Screen Shot 2019-10-11 at 17 21 22" src="https://user-images.githubusercontent.com/5032900/66659383-14137580-ec4c-11e9-918d-d786adae71c0.png">

<img width="375" alt="Screen Shot 2019-10-11 at 17 21 36" src="https://user-images.githubusercontent.com/5032900/66659384-14ac0c00-ec4c-11e9-8923-b143e3388448.png">

Reply:
I was thinking the same thing about the Actions. They are separate, so they can probably be applied "external" to the groupings/settings that we are sharing with web, aka a "mobile add-on" group.

Reply:
Thanks for a feedback. Make sense to me to create a specific container for action buttons as well, if you agree on that I will refactor my PR.

Reply:
I'd say yes let's do that but I am not %100 sure about this being just a container 🤔 It can rather be a component that takes array of {title, onPress} tuples and handle the rendering of buttons inside. Actually, you can do what you think is the best and we can iterate on that. Thanks!

Reply:
I've refactored a PR and will update the description according to new changes.